swagger.json

```diff
@@ -6257,6 +6474,117 @@
             "in": "query",
             "type": "integer",
             "format": "int64"
+        },
+        "createUserParam": {
+            "name": "user",
+            "description": "User properties. Only one of organizationRole or tagRoles is required.",
+            "required": true,
+            "in": "body",
+            "schema": {
+                "type": "object",
+                "required": [
+                    "name",
+                    "email",
+                    "authType"
+                ],
+                "properties": {
+                    "name" : {
+                        "type": "string",
+                        "description": "The first and last name of the user.",
+                        "example": "Jane Doe"
+                    },
+                    "email": {
+                        "type": "string",
+                        "description": "The email address of this user.",
+                        "example": "user@company.com"
+                    },
+                    "organizationRole" : {
+                        "type": "string",
+                        "description": "The name of the role the user is assigned to for the organization. Leave this blank to give only tag access. This must match one of the preset roles or the name of a custom role in the organization. This value is case insensitive.",
+                        "example": "Read-Only Admin"
+                    },
+                    "tagRoles" : {
+                        "type": "array",
+                        "description": "If specified, the user will be a tag administrator for these tags. If left blank, the user has access to the entire organization.",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "tagName": {
```

Comment:
We allow tags to be referenced by name during csv upload so I am proposing we do the same here.